objc/UIViewController+PromiseKit.m

```diff
@@ -6,11 +6,13 @@
 #import <UIKit/UINavigationController.h>
 #import <UIKit/UIImagePickerController.h>
 #import "UIViewController+PromiseKit.h"
+@import MessageUI.MFMessageComposeViewController;
+@import MessageUI.MFMailComposeViewController;
```

Comment:
Yes, that does suck. Well it's ok to hard code the int values, they won't change. Apple won't risk it. but it is sucky.

For PMK2 I am changing this sort of stuff so we can @import, by making the end user compile the categories into their own end binaries. I have an easy way to facilitate that so it won't be a PITA.